src/scenic/core/regions.py

```diff
@@ -2397,11 +2397,21 @@ class PathRegion(Region):
     Args:
         points: A list of points defining a single polyline.
         polylines: A list of list of points, defining multiple polylines.
+        orientation:
         tolerance: Tolerance used internally.
     """
 
-    def __init__(self, points=None, polylines=None, tolerance=1e-8, name=None):
-        super().__init__(name)
+    def __init__(
+        self, points=None, polylines=None, tolerance=1e-8, orientation=True, name=None
+    ):
+        if orientation is True:
+            orientation = VectorField("Path", self.defaultOrientation)
+            self.usingDefaultOrientation = True
+        else:
+            self.usingDefaultOrientation = False
```

Comment:
Probably better to use `_usingDefaultOrientation` since it's an implementation detail.

Reply:
I copied this over from PolyLine region, should I swap it there as well?

Reply:
Yes. `PolylineRegion` dates from very early days when we had minimal documentation and I wasn't careful about this sort of thing.